arch/xtensa/src/esp32s3/hardware/esp32s3_rtccntl.h

```diff
@@ -0,0 +1,5795 @@
+/****************************************************************************
+ * arch/xtensa/src/esp32s3/hardware/esp32s3_rtccntl.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_RTCCNTL_H
+#define __ARCH_XTENSA_SRC_ESP32S3_HARDWARE_ESP32S3_RTCCNTL_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include "esp32s3_soc.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* The value that needs to be written to RTC_CNTL_WDT_WKEY to
+ * write-enable the wdt registers
+ */
+
+#define RTC_CNTL_WDT_WKEY_VALUE     0x50d83aa1
+
+/* RTC_CNTL_RTC_OPTIONS0_REG register
+ * RTC common configure register
+ */
+
+#define RTC_CNTL_RTC_OPTIONS0_REG (DR_REG_RTCCNTL_BASE + 0x0)
+
+/* RTC_CNTL_SW_SYS_RST : WO; bitpos: [31]; default: 0;
+ * SW system reset
+ */
+
+#define RTC_CNTL_SW_SYS_RST    (BIT(31))
```

Comment:
minor
```suggestion
#define RTC_CNTL_SW_SYS_RST    BIT(31)
```
here and in other places